tools/depends/configure.ac

```diff
@@ -171,21 +172,21 @@ case $build in
   *darwin*)
     build_os="osx"
     if test "x$build_cpu" = "xx86_64"; then
-      host_includes=-mmacosx-version-min=10.13
+      host_includes="-arch x86_64 -mmacosx-version-min=10.13"
     else
-      host_includes=-mmacosx-version-min=11.0
+      host_includes="-arch arm64 -mmacosx-version-min=11.0"
     fi
-    use_xcodepath=`xcode-select -print-path`
-    use_build_toolchain=$use_xcodepath
-    use_xcodebuild=$use_xcodepath/usr/bin/xcodebuild
-    AC_MSG_RESULT(found xcodebuild at $use_xcodebuild)
+    use_xcrun=xcrun
 
     # acquire build platform (native) sdk sysroot.
     build_platform=macosx
-    native_sdk_path=[`$use_xcodebuild -version -sdk $build_platform | grep ^Path | awk '{ print $2}'`]
+    native_sdk_path=[`$use_xcrun --show-sdk-path`]
     host_includes+=" -isysroot $native_sdk_path"
 
-    android_toolchain_name="darwin-x86_64"
+    CC_FOR_BUILD=[`$use_xcrun -find clang`]
```

Comment:
should be either `-f` or `--find` per `xcrun`'s manual, there's no `-find` (single dash version)

Reply:
Odd, it seems to work on Monterey. I'll go back and have a look at it

Reply:
single dash operators all worked. Ive changed all to match double dash though just for uniformity of documented man